modules/claims_api/spec/fixtures/v2/veterans/disability_compensation/form_526_json_api.json

```diff
@@ -177,7 +177,7 @@
         },
         "federalActivation": {
           "activationDate": "2023-10-01",
-          "anticipatedSeparationDate": "2024-10-31"
+          "anticipatedSeparationDate": "2025-10-31"
```

Comment:
we should try to find a way to make this dynamic since it has to be future, or we'll update every year. Something like `6.months.from_now.strftime('%F')` - this can be another PR / task though